cpp/ps0004/cpp_ps0004.cpp

```diff
@@ -1,57 +1,39 @@
-/* ps0004 - Give me a number! I'll send back a character.
-=========================================================
-Write a program that takes a number as input and outputs an equivalent ASCII Character.
-
-- If the inputted data is a zero-value, a character, a letter or a text, print
-      "Invalid character/s or zero value!" and exit the program RETURNING 1.
-
-- If the inputted data is a floating number, print
-      "Invalid: a floating point number!" and exit the program RETURNING 2.
-
-- If the inputted data is a negative or out-of-range number, print
-      "Invalid: number is out of the range!" and exit the program RETURNING 3.
-
-- If the inputted data is a number [32:126], print
-      "The character for <PRINT_YOUR_NUMBER_HERE> is <PRINT_THE_EQUIVALENT_CHARACTER_HERE>." and exit the program RETURNING 0.
-*/
-
 #include <iostream>
 #include <cmath>
 
 
 int main()
 {
-      // prompting user for the number---------------------------------------
+      // prompting user for the number--------------------------------------- 1.5/1.5
       float inputValue {};
       std::cout << "Enter a number from 32 to 126 : " << std::endl;
       std::cin >> inputValue;
 
       short int convertedValue = inputValue;

```

Comment:
- [x] declared a variable to store the user input
- [x] chose a suitable data type for the variable
- [x] prompted user to enter a number
- [x] used `std::cin` in a right way to get user's input